Core/Security/ProxyInfo.py

```diff
@@ -155,7 +155,7 @@ def formatProxyStepsInfoAsString(infoList):
       if key in stepInfo:
         value = stepInfo[key]
         if key == 'serial':
-          value = base64.b16encode(value)
+          value = base64.b16encode("%s" % value)
```

Comment:
Can you add a comment? Without, I see a future me changing it back to what it was.

Reply:
done